common/src/main/java/bisq/common/app/Capability.java

```diff
@@ -31,7 +31,7 @@
     PROPOSAL,
     BLIND_VOTE,
     ACK_MSG,
-    BSQ_BLOCK,
+    RECEIVE_BSQ_BLOCK,
```

Comment:
I am not sure we should change stuff like that. What I am sure of is that we should add Javadoc to describe what a capability is for and what it does, only so can we make sure devs can understand what these 2 or 3 words mean.